Documentation/ApiOverview/Authentication/CSRFlikeRequestTokenHandling.rst

```diff
@@ -0,0 +1,163 @@
+..  include:: /Includes.rst.txt
+..  index:: Authentication, Request token
+..  _authentication-request-token:
+
+================================
+CSRF-like request token handling
+================================
+
+..  versionadded:: 12.0
+
+A CSRF-like request token handling is available to mitigate potential cross-site
+requests on actions with side effects. This approach does not require an
+existing server-side user session, but uses a nonce (number used once) as a
+"pre-session". The main scope is to ensure a user actually has visited a page,
+before submitting data to the webserver.
+
+This token can only be used for HTTP methods `POST`, `PUT` or `PATCH`, but
+for instance not for `GET` request.
+
+The :php:`\TYPO3\CMS\Core\Middleware\RequestTokenMiddleware` resolves
+request tokens and nonce values from a request and enhances responses with
+a nonce value in case the underlying application issues one. Both items are
+serialized as JSON Web Token (JWT) hash signed with `HS256`. Request tokens
```

Comment:
Do readers have to understand these technological deatails? If so I would link somewhere for additional information

Reply:
I assume, if someone wants to create a custom authentication, they should know what they are doing and are familiar with these concepts. I consider this chapter as "for specialists".

Reply:
But JWT is linked.